packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -15,7 +15,6 @@
 
   .layout {
     display: flex;
-    font-family: 'Roboto', sans-serif;
```

Comment:
this is non passive. you will break healthe at cerner doing this. can we set a default of roboto and allow memorial herman to override this? 
@mhemesath thoughts? should the library just not set this and require healthe@cerner to set this to roboto (as well as portal)

Reply:
I am importing terra-base's styles in Layout.jsx as suggested by @bjankord and @mhemesath . Should I still keep this line?

Reply:
@alex-bezek the point is to update healthe-at-cerner to configure the theme at the application level

Reply:
we discussed, this makes sense. just make sure you update the themes that were relying on this. I'll probably release this as a non passive change and bump the major version